apps/storefront/src/pages/QuoteDraft/index.test.tsx

```diff
@@ -1275,6 +1331,340 @@ describe('when the user is a B2B customer', () => {
       expect(within(productTable).queryByText('Insufficient stock')).not.toBeInTheDocument();
     });
 
+    it('does not call the function getVariantInfoOOSAndPurchase', async () => {
+      set(window, 'b2b.callbacks.dispatchEvent', vi.fn().mockReturnValue(true));
+      vi.mock('uuid', () => ({
+        v4: vi.fn(() => '1d233-34343a'),
+      }));
+
+      const getVariantInfoOOSAndPurchase = vi.fn();
```

Comment:
This is defined, never used and check for `.not.toHaveBeenCalled()`, it can be removed.